cilium-cli/cli/clustermesh.go

```diff
@@ -322,13 +322,17 @@ func newCmdClusterMeshDisconnectWithHelm() *cobra.Command {
 			}
 		},
 	}
+	cmd.Flags().StringVar(&params.ConnectionMode, "connection-mode", "bidirectional", "Connection mode: unicast, bidirectional or mesh")
 	cmd.Flags().StringVar(&params.DestinationContext, "destination-context", "", "Kubernetes configuration context of destination cluster")
+	cmd.Flags().StringSliceVar(&params.DestinationContexts, "destination-contexts", []string{}, "Comma separated list of Kubernetes configuration contexts of destination cluster")
```

Comment:
This seems a bit confusing ie is destination-context still valid or is it destination-contexts=xxx? 
Not sure we need to keep both options??

Reply:
For retro-compatibility, I kept destination-context. You can use both, it just add to the destination-contexts list. But if you think that the destination-context option is now useless. I can remove this option.